Loenn/entities/lobbyMapWarp.lua

```diff
@@ -0,0 +1,46 @@
+local drawableSprite = require("structs.drawable_sprite")
+
+local lobbyMapWarp = {}
+lobbyMapWarp.name = "CollabUtils2/LobbyMapWarp"
+lobbyMapWarp.fieldInformation = {
+    playerFacing = {
+        fieldType = "integer",
+        options = {
+            { "Left", -1 }, { "Right", 1 }
```

Comment:
you can also use the [celesteEnums](https://github.com/CelestialCartographers/Loenn/blob/master/src/consts/celeste_enums.lua) include for stuff like this